src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetPeerInfoResponseTests.cs

```diff
@@ -46,7 +46,7 @@ public void GetPeerInfoResponse_Can_Get_Output()
             getPeerInfoResponse.PeerInfo.Add(peer);
 
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getPeerInfoCommand = new GetPeerInfoCommand(commandContext, Substitute.For<ILogger>());
+            new GetPeerInfoCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'GetPeerInfoCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)